doc/widget-api-frontend.md

```diff
@@ -62,42 +62,7 @@ contentfulWidget.init(function (widget) {
 ## `widget.contentType`
 
 This API gives you access to data about the content type and the entry.
-It has the following shape.
-~~~js
-{
-  "name": "<content type name>",
-  "sys": {
-    "space": {
-      "sys": {
-        "type": "Link",
-        "linkType": "Space",
-        "id": "<space ID>"
-      }
-    },
-    "id": "<content type ID>",
-    "type": "ContentType",
-    "createdAt": "<creation date>",
-    "updatedAt": "<updation date>",
-    "revision": <revision number>
-  },
-  "displayField": "<id of display field>",
-  "fields": [<collection of fields in entry>]
-}
-~~~
-
-Each field in `fields` collection can have the following shape.
-~~~js
-{
-  "id": "<id of field>",
-  "name": "<name of field>",
-  "type": "<type of field. e.g., Symbol, Text, etc>",
-  "localized": <boolean>,
-  "required": <boolean>,
-  "validations": [<validations>],
-  "disabled": <boolean>,
-  "omitted": <boolean>
-}
-~~~
+It has the shape as described under "content type properties" [here](https://www.contentful.com/developers/docs/references/content-management-api/#/reference/content-types).
```

Comment:
Please don’t use “here” link as [this Smashing Mag post](https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/) explains. Also applies to `widget.field.type` section